bin/collator/src/parachain/service.rs

```diff
@@ -64,18 +67,9 @@ use crate::{
 };
 
 /// Extra host functions
-#[cfg(feature = "runtime-benchmarks")]
```

Comment:
@Dinonard I think we need this

Reply:
how is the benchmarking::HostFunctions injected?

Reply:
nevermind, cli will add it https://github.com/paritytech/polkadot-sdk/blob/8c8edacf8942298c3807a2e192860da9e7e4996a/substrate/utils/frame/benchmarking-cli/src/pallet/command.rs#L247